UnitsNet/QuantityInfo.cs

```diff
@@ -21,52 +20,28 @@ namespace UnitsNet
     /// </remarks>
     public class QuantityInfo
     {
-        private static readonly string UnitEnumNamespace = typeof(LengthUnit).Namespace;
-
-        private static readonly Type[] UnitEnumTypes = typeof(Length)
-            .Wrap()
-            .Assembly
-            .GetExportedTypes()
-            .Where(t => t.Wrap().IsEnum && t.Namespace == UnitEnumNamespace && t.Name.EndsWith("Unit"))
-            .ToArray();
-
-        /// <summary>
-        ///     Constructs an instance.
-        /// </summary>
-        /// <param name="quantityType">The quantity enum value.</param>
-        /// <param name="unitInfos">The information about the units for this quantity.</param>
-        /// <param name="baseUnit">The base unit enum value.</param>
-        /// <param name="zero">The zero quantity.</param>
-        /// <param name="baseDimensions">The base dimensions of the quantity.</param>
-        /// <exception cref="ArgumentException">Quantity type can not be undefined.</exception>
-        /// <exception cref="ArgumentNullException">If units -or- baseUnit -or- zero -or- baseDimensions is null.</exception>
-        [Obsolete("QuantityType will be removed in the future. Use QuantityInfo(string, UnitInfo[], Enum, IQuantity, BaseDimensions) instead.")]
-        public QuantityInfo(QuantityType quantityType, [NotNull] UnitInfo[] unitInfos, [NotNull] Enum baseUnit, [NotNull] IQuantity zero, [NotNull] BaseDimensions baseDimensions)
-            : this(quantityType.ToString(), unitInfos, baseUnit, zero, baseDimensions, quantityType)
-        {
-        }
-
         /// <summary>
         ///     Constructs an instance.
         /// </summary>
         /// <param name="name">Name of the quantity.</param>
+        /// <param name="unitType">The unit enum type, such as <see cref="LengthUnit" />.</param>
         /// <param name="unitInfos">The information about the units for this quantity.</param>
         /// <param name="baseUnit">The base unit enum value.</param>
         /// <param name="zero">The zero quantity.</param>
         /// <param name="baseDimensions">The base dimensions of the quantity.</param>
         /// <param name="quantityType">The the quantity type. Defaults to Undefined.</param>
         /// <exception cref="ArgumentException">Quantity type can not be undefined.</exception>
         /// <exception cref="ArgumentNullException">If units -or- baseUnit -or- zero -or- baseDimensions is null.</exception>
-        public QuantityInfo([NotNull] string name, [NotNull] UnitInfo[] unitInfos, [NotNull] Enum baseUnit, [NotNull] IQuantity zero, [NotNull] BaseDimensions baseDimensions,
```

Comment:
This would also be a breaking change, however since it was recently introduced and it didn't actually work as intended, then I'm ok with changing it.